README.md

```diff
@@ -0,0 +1,99 @@
+# Portfolio Project
+
+> This is a project on building my personal portfolio
+
+![screenshot](./img/preview.png)
+
+This is the first project i will carry out on HTML and CSS and encompasses all of the major functions and purpose of HTML and CSS, also creating a platform to show my works. It is based on a mobile first approach.
+
+## Built With
+
+- HTML
+
+  - Unordered List (To list out certain items on the page)
+  - nav element (To create a nav-bar)
+  - a tags (To link to other external webpages)
+  - svg are also used in the HTML
```

Comment:
I would use code syntax e.g. `svg`, `a`, `nav` to represent the elements mentioned, this way, they are not confused with normal words.